nextflow.config

```diff
@@ -20,7 +20,7 @@ params {
     diginorm_C                  = 20
     diginorm_k                  = 20
     orf_caller                  = null
-
+    
```

Comment:
I think this (space characters at the end of the line) will fail in linting, so remove the spaces. Also below, line 41.